person.rb

```diff
@@ -0,0 +1,24 @@
+# Person: A class to define attributes and behaviors
+class Person
+  # Constructor to initialize the class Person
+  def initialize(age, name = 'Unknown', parent_permission = 'true')
+    @id = "#{Random.rand(1..1000)}-#{name}"
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  # accessor get and set method
+  attr_accessor :name, :age
+  attr_reader :id
```

Comment:
- [ ] [OPTIONAL] It is best practice to write getter &  setter attributes after starting class. It should be written after `class Person`, kindly move it up.

Reply:
## Hi @Deepakdanger,
Thanks for the tip, I'll implement the change now.